CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 - Plugin interface for HTML preprocessor
 - Plugin interface for HTML postprocessor
 - Plugin module with `[video]`, `[audio]` and `[caption]` WordPress shrotcode support
+- Added OnMediaDeletedListener interface, detection and handling
```

Comment:
Excellent, you remembered the changelog! :) Just a nitpick - I've been leaving out the verb for clarity (No "Added", "Fixed", "Changed), since it's already obvious by the section title.